<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MEMBERS PAGE</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MEMBERS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>16</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="membersTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1361</width>
     <height>621</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <column>
    <property name="text">
     <string>INDEX</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>FULL NAME</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>NATIONAL ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DATE JOINED</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ROLE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ADDRESS</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>STATUS</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="go_to_register">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REGISTER NEW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_member_btn">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ADD MEMBER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
